### PR TITLE
Mauriciogtec/issue14

### PR DESCRIPTION
**Pull request by mauriciogtec**
- created data/make_dataset.py
- data/data.sh now downloads this file.
- removed the script portion from space/dataset.py

### CODE REVIEW

File: data/data.sh [hunk 1]
```diff
@@ -17,14 +17,20 @@ mkdir .dataset_downloads
...
 
-mv counties.geojson $user_path
-mv counties.graphml $user_path
+# if user path is not .dataset_downloads, move the data to the user path
+if [ "$user_path" != ".dataset_downloads" ]; then
+    mv counties.geojson $user_path
+    mv counties.graphml $user_path
+    cd ..
+    rm -rf .dataset_downloads
+fi
```
Comment (atrisovic) [inline data/data.sh:33]: This should not be here because `make_dataset.py` will save the file at $user_path (or local path if $user_path is none).

**Review by atrisovic — CHANGES_REQUESTED** [review]

**Comment by mauriciogtec** [comment]
The code under `if [ "$user_path" != ".dataset_downloads" ]` is needed to consider the edge case when the user selects "$user_path" != ".dataset_downloads". Since there was a bug where everything was deleted after download due to using `rm <...>`.

**Comment by audiracmichelle** [comment]
I think this specific PR should go into master because that branch will remain exclusively for readers of the CLeaR submission, correct?
all other spacebench package related PR will go to dev and then main? is this correct @Naeemkh ?

**Comment by Naeemkh** [comment]
> I think this specific PR should go into master because that branch will remain exclusively for readers of the CLeaR submission, correct? all other spacebench package related PR will go to dev and then main? is this correct @Naeemkh ?

Yes. If it is related to old submission, it is better to be part of master branch. When you are done with that submission, we will drop the master branch and use the main branch.  @mauriciogtec, you can change the PR base branch.

**Pull request merged**